### PR TITLE
Point README at production acme endpoint

### PR DESCRIPTION
**Pull request by denniscollective**
:lipstick: :rocket:

### CODE REVIEW

**Comment by unixcharles** [comment]
Now that its letsencrypt is live! :+1:

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ require 'openssl'
 private_key = OpenSSL::PKey::RSA.new(2048)
 
 # We need an ACME server to talk to, see github.com/letsencrypt/boulder
-endpoint = 'https://acme-staging.api.letsencrypt.org'
+endpoint = ''https://acme-v01.api.letsencrypt.org/'
```
Comment (unixcharles) [inline README.md]: oops, will fix the typo

Comment (denniscollective) [inline README.md]: OOOPPPSS sorry about that! <3

Comment (unixcharles) [inline README.md]: no worry

**Pull request merged**